util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtil.java

```diff
@@ -0,0 +1,219 @@
+package com.google.cloud.healthcare.imaging.dicomadapter;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.dcm4che3.data.Attributes;
+import org.dcm4che3.data.Sequence;
+import org.dcm4che3.data.Tag;
+import org.dcm4che3.data.VR;
+import org.dcm4che3.net.Status;
+import org.dcm4che3.net.service.DicomServiceException;
+import org.dcm4che3.util.TagUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class AttributesUtil {
+
+  public static String getTagValue(JSONObject json, String tag) throws JSONException {
```

Comment:
This method doesn't seem to be used, can we delete it?

Reply:
It it used in next pull request, for cases where not having data should throw exceptions.

Reply:
After looking in more detail: there is a copy of this method in export adapter CStoreSender. Removed it to use AttributesUtil one instead, thus also solving problem of unused code.